apps/web/lib/features/integrations/jitsi.tsx

```diff
@@ -0,0 +1,42 @@
+import { JITSI_DOMAIN } from '@app/constants';
+import { useOrganizationTeams } from '@app/hooks';
+import { JitsiMeeting } from '@jitsi/react-sdk';
+import { useRouter } from 'next/router';
+
+export default function CallPage({ jwt }: { jwt: string }) {
+	const { activeTeam } = useOrganizationTeams();
+	const router = useRouter();
+
+	const roomName = activeTeam?.name
+		.toLowerCase()
+		.replace(/(?<= )[^\s]|^./g, (a) => a.toUpperCase())
+		.replaceAll(' ', '');
+
+	return (
+		<JitsiMeeting
+			domain={JITSI_DOMAIN}
+			roomName={'EverTeam' + roomName}
+			configOverwrite={{
+				startWithAudioMuted: true,
+				startWithVideoMuted: true,
+				disableModeratorIndicator: false,
```

Comment:
@paradoxe35 any reason why moderator indicator disabled?
Also, `startScreenSharing`, does it mean that when video started it automatically start screen sharing if that option enabled (if so, let's keep it false).